### DISCUSSION TITLE
What is the benefit of Prisma's `relationMode`?

### DISCUSSION THREAD

**Discussion by hckhanh** [prisma/prisma#16534] [Q&A] [answered]
Hi guys, I', just curious about the new Prisma feature.
Is there any document that explains more details about `relationMode` and its benefits?

**Comment by bn-piar** [comment, marked as answer]
Hi @hckhanh. There's a page about relation modes https://www.prisma.io/docs/concepts/components/prisma-schema/relations/relation-mode

**Reply by janpio** [reply]
Does that explain it well enough @hckhanh?

**Reply by vladi-stevanovic** [reply]
:wave: Hello @hckhanh , I’m going through pending GitHub Discussions to check if you need any help. Is this issue still open?

Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! :raised_hands:

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
